modules/round/src/main/JsonView.scala

```diff
@@ -356,10 +356,10 @@ object JsonView {
     import lila.common.Maths.truncateAt
     Json.obj(
       "running" -> c.isRunning,
-      "initial" -> c.limit,
-      "increment" -> c.increment,
-      "white" -> truncateAt(c.remainingTime(Color.White), 2),
-      "black" -> truncateAt(c.remainingTime(Color.Black), 2),
+      "initial" -> c.limitSeconds,
+      "increment" -> c.incrementSeconds,
+      "white" -> c.remainingTime(Color.White).toSeconds,
+      "black" -> c.remainingTime(Color.Black).toSeconds,
```

Comment:
the only safe ways to do that which I can think of are:
a) custom float serializer which writes only 2 decimals
b) pass int centis instead of a float

the current scheme will still usually only have 2 decimals, since it's the result of `(int / 100f)`, so I think this is OK as is.